lamby/src/config.py

```diff
@@ -0,0 +1,88 @@
+import click
+import sys
+from tempfile import mkstemp
+from shutil import move
+import os
+from os import fdopen
+
+
+@click.command('config', short_help='modify configuration parameters')
+@click.option('--add', nargs=2, help='usage: --add [key] [value] — adds' +
+              ' key/value pair to config file')
+@click.option('--change', nargs=2, help='usage: --change [key] [value] ' +
+              '— changes key to have given value in config file')
+@click.option('--remove', nargs=1, help='usage: --remove [key] ' +
+              '— removes the selected key from the config file')
+def config(add, change, remove):
+    """Modify configuration parameters for this initialized Lamby project"""
+
+    lamby_dir = './.lamby'
+    if not os.path.isdir(lamby_dir):  # Check if current dir has .lamby file
+        click.echo('Lamby project not initialized in ' + os.getcwd())
+        sys.exit(1)
+
+    # add #
+
+    if len(add) != 0:
+        # See if key already exists #
+        with open(lamby_dir + '/config', "r") as config_file:
+            for line in config_file:
```

Comment:
Use src.utils deserialize_log and serialize_log to save it back. It will parse the JSON into a python dictionary.

Reply:
I will do this. I made it non-JSON at first following the GitHub format, but I think using a JSON format should work just as well